views/header.html

```diff
@@ -7,20 +7,18 @@
 });
 </script>
 <div class="header" >
-  <div class="container">
-    <h4 class="mast">FreeCodeCamp Indy</h4>
-    <div class="rightSide">
-      <ul class="headList">
-        <li> <a href="index.html">Home</a> </li>
-        <li> <a href="info.html">Info</a> </li>
-        <li> <a href="campers.html">Campers</a></li>
-        <li> <a href="gallery.html">Gallery</a></li>
-        <li> <a href="contact.html">Contact</a></li>
-      </ul>
-    </div>
-    <div class="hamburger">
-      <button class="menuBtn ">Menu</button>
-    </div>
+  <h4 class="mast">FreeCodeCamp Indy</h4>
+  <div class="rightSide">
+    <ul class="headList">
+      <li> <a href="index.html">Home</a> </li>
```

Comment:
Remove the extra space between `<li>` and `<a`  and also from between `</a>` and `</li>`.

The extra space can cause differences in the distances between the content around this `<li>`. These distances should be controlled by `margin` or `padding` in CSS.